### ISSUE TITLE
openshot does not accept files

### ISSUE THREAD

**Issue by lodar1** [OpenShot/openshot-qt#4573]
**Describe the bug**
It is not possible to import a file (mp4, mov, Mp3 ...)
Neither with a green cross nor with drag and drop

(but there are no other problems on the PC-system!) 

**Steps to reproduce the behavior:**
I click on the green circle and choose a file. But nothing happens.
At the bottom of the screen there is a brief message: 0 files imported 
I click and hold a video or music file with the left mouse button (dateiexplorer) and drag it into the project data field while holding the button down. Here, too, the same result: 0 files ..... 

**Expected behavior**
Either way, I expected the files to be added to "Project data". 

**System Details**
- OpenShot Version: 2.6.1-dev
- libopenshot Version:  0.2.7-dev
- Platform: Windows-10-10.0.19041
- Processor: AMD64 Family 23 Model 17 Stepping 0, AuthenticAMD
- Machine: AMD64
- Python version: 3.8.9
- Qt5 version: 5.15.2
- PyQt5 version: 5.15.4
- Qt Detected Languages: ['de-DE', 'en-US']
- LANG Environment Variable: 
- LOCALE Environment Variable: 
- Daily Build: Verified issue still exists in daily build: http://github.com/OpenShot/openshot-qt/releases/download/OpenShot-v2.6.1-dev-daily-8803-18bdad43-0fb4ab09-x86_64.exe

**Log Files**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/nw482d2y9so7n67qeb2w5348z6alhkz6/openshot-qt.zip) (11 KB)

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
If applicable, add screenshots to help explain your problem. You can include screenshots by
copy/pasting them on GitHub or dragging-and-dropping into the GitHub page. All images are public,
so please don't post screenshots containing personal information.

**Comment by JacksonRG** [comment]
Makes me wonder if you have permission to access the files. Are the files located in another user's drive? Or have you tried launching the program as an administrator?

But that's just my first thought. Taking a look at your logs. :mag:

**Comment by lodar1** [comment]
Hello,


I have full access to the files.
Tried with and without administrator rights.


I can't see any logs, the program is currently deleted.


Here is what has already been tried:

 




Even a project doesn't change anything.
The only difference is that the mouse pointer is no longer a red circle, but the file image.
But when I let go, nothing is added.


With me everything is allowed, except special authorizations.With all 4 users.

I installed Windows 64 bit and also the 64 bit version, but I'm trying anyway.

With 32 bit is the same Problem.


Nothing happens both times, the mouse arrow always remains a red circle with a diagonal.
Click and hold the file with the left mouse button, then drag it from Explorer to the "Project files" area with the mouse button depressed. The mouse pointer turns into a circle with a diagonal line.
Or click on the large green cross and double-click on file in the Explorer that opens.
I tried that with an Mp3, no success.rmesdjian: I followed your instructions, but the same result again. Cannot insert a file.
Now I have completely uninstalled the program again and reinstalled it with administrator rights.
No change


I wrote the following a couple of hours ago, but it probably didn't work:
to questions from rmesdjian:


	
	Drag the file into Open Shot and import files in two ways.
	namely with .mov and .mp4
	
	
	Message at the bottom of the screen: 0 files are being imported
	
	
	see point 2
	
	
	Windows 10
	
	
	2.6.1
	





 
 

Gesendet: Donnerstag, 09. Dezember 2021 um 00:34 Uhr
Von: "JacksonRG" ***@***.***>
An: "OpenShot/openshot-qt" ***@***.***>
Cc: "lodar1" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [OpenShot/openshot-qt] openshot does not accept files (Issue #4573)


 

Makes me wonder if you have permission to access the files. Are the files located in another user's drive? Or have you tried launching the program as an administrator?

But that's just my first thought. Taking a look at your logs. 🔍

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
